scss/components/_activity-month-nav.scss

```diff
@@ -0,0 +1,37 @@
+.civicase__activity-month-nav__group {
+  border-left: 2px solid $activity-month-nav-divider;
+
+  [class^='civicase__activity-month-nav__group-'] {
+    padding-left: 15px;
```

Comment:
So we set padding-left: 15px to the parent and use them as vars to calculate left margin for active text
```
.civicase__activity-month-nav__group {
  $month-nav-border-width: 2px;
  $month-nav-left-spacing: 15px;

  border-left: $month-nav-border-width solid $activity-month-nav-divider;
  padding-left: $month-nav-left-spacing;  

  ...
  ...

  &.active {
    margin-left: -1 * ($month-nav-border-width + $month-nav-left-spacing)
    ...
    ...
  }
}
```

or either use a common class to all the children elements of `.civicase__activity-month-nav__group `. Whatever seems right ?